crates/api/src/community/ban.rs

```diff
@@ -46,6 +50,9 @@ impl Perform for BanFromCommunity {
 
     // Verify that only mods or admins can ban
     is_mod_or_admin(context.pool(), local_user_view.person.id, community_id).await?;
+    if let Some(reason) = &data.reason {
+      is_valid_body_field(reason)?;
+    }
```

Comment:
As most of these are optional, I think it would be cleaner if the functions would also take `Option<String>` as param.

Reply:
I thought about that, but there's a lot of places where the bodies are possibly `Option<Option<String>>`, so it seemed best to leave it outside.

Reply:
Im not seeing `Option<Option<String>>` anywhere. And there are only a few cases without option, there it would work to pass the parameter as `Some(body)`.